discussion/about-discussion.md

```diff
@@ -33,26 +31,26 @@ If you plan on using a whole lot of terms that might not be familiar to your rea
 
 Declaring them here lets you use any abbreviated forms of the terminology you're using in the body of the discussion
 
-### Structuring Discussion Articles
+### Structuring discussion articles
 
 Think about structuring discussion topics like you would a presentation to a group of folks who don't know anything about your chosen subject.
 Your discussion should introduce ideas gradually so your audience can grow their understanding of the concepts in your discussion.
 
-It can be easy to blend Discussions with other types of content, like How-to or Reference articles.
+It can be easy to blend discussions with other types of content, like How-to or Reference articles.
```

Comment:
If you are setting "discussions" to lowercase here you probably should keep it consistent with "how-to" and "reference" as well.